docs/source/item-validation.rst

```diff
@@ -234,20 +234,43 @@ A `list` containing the local path of the item schema.
 
     # settings.py
 
-    SPIDERMON_VALIDATION_CERBERUS: [
+    SPIDERMON_VALIDATION_CERBERUS = [
         '/path/to/schema.json',
         'http://example.com/mycerberusschema',
         {"Field": {"type": "number", "required":True}}
     ]
 
-If you are working on a spider that produces multiple items types, you can define it
-as a `dict`:
+If you are working on a spider that produces multiple items types, you can define paths to schema for each item as `dict` like the example below:
```

Comment:
I am not sure if I liked this example. It doesn't look like a real-world scenario where we return more than one item. I would remove it and leave only the `SPIDERMON_VALIDATION_CERBERUS` code (the same way that the other validation libraries is presented).

Reply:
Roger that, we can think of something else in the tutorial we are building. 
Resolving.